src/graphics/program-lib/chunks/skinTex.vert

```diff
@@ -1,8 +1,8 @@
-attribute vec4 vertex_boneWeights;
-attribute vec4 vertex_boneIndices;
+attribute highp vec4 vertex_boneWeights;
+attribute highp vec4 vertex_boneIndices;
```

Comment:
What happens on devices that don't support hiphp?

Reply:
we only use those chunks when float textures are support, which should mean highp is supported.

Reply:
Isn't it just the `texture_poseMap` that needs to be `highp`?